lessons/Early-Math/distributive-property/langs/en-us/pages/mental-math.adoc

```diff
@@ -0,0 +1,54 @@
+= The Distributive Property and Mental Math
+
+On this worksheet, the goal is to *make the math easier* by creating equivalent Circles so that we can eventually solve in our heads. To do so, fill in each Circle, moving from left to right. Then, use mental math to compute the answer. The first one is done for you.
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+
+
+[.FillVerticalSpace,cols=".^1a,^.^3a,^.^7a,^.^10a,^.^2a", stripes="none", options="header"]
+|===
+|	 | Numeric Expression | Product | Sum or Difference | Answer
+
+| @n
+| @show{(coe '(* 70 39))}
+| @show{(coe '(* 70 (- 40 1)))}
+| @show{(coe '(- (* 70 40) (* 70 1)))}
+| @math{210}
+
+| @n
+| @show{(coe '(* 20 29))}
+| @show{(coe '(* (?ANS 20) (- (?ANS 30) (?ANS 1))))}
+| @show{(coe '(- (* (?ANS 20) (?ANS 30)) (* (?ANS 20) (?ANS 1))))}
+| @ifsoln{@math{580}}
+
+| @n
+| @show{(coe '(* 50 49))}
```

Comment:
let's change this one to 50 X 51. so you've practiced both adding and subtracting with 1 before switching distance.